project-management/src/main/java/life/qbic/projectmanagement/domain/model/experiment/vocabulary/OntologyClassDTO.java

```diff
@@ -0,0 +1,119 @@
+package life.qbic.projectmanagement.domain.model.experiment.vocabulary;
+
+import java.util.Objects;
+import life.qbic.projectmanagement.application.OntologyClassEntity;
+
+public class OntologyClassDTO {
+
+  String ontology;
+
```

Comment:
Since this is a key DTO in our application some JD would be good to explain it's usage and context 🥇